### PR TITLE
Add Mumbling Kata to Katas

### PR DESCRIPTION
**Pull request by dushan-madetech**
Add the Mumbling Kata to Learn Tech Katas based on the Mumbling Kata found on [Codewars](https://www.codewars.com/kata/mumbling)

### CODE REVIEW

File: katas/mumbling/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Mumbling Kata
+
+The goal of this kata is to implement the `accum()` method which takes a string as input and returns a formatted output string.
+
+The following examples illustrate the `accum()` method:
+
+```ruby
+accum("a") = "A"
```
Comment (craigjbass) [inline katas/mumbling/README.md]: Did you mean
```suggestion
accum("a") == "A"
```

File: katas/mumbling/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Mumbling Kata
+
+The goal of this kata is to implement the `accum()` method which takes a string as input and returns a formatted output string.
+
+The following examples illustrate the `accum()` method:
```
Comment (craigjbass) [inline katas/mumbling/README.md]: Don't like `accum` - would suggest we come up with a better method name on our own repo

File: katas/mumbling/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Mumbling Kata
+
+The goal of this kata is to implement the `accum()` method which takes a string as input and returns a formatted output string.
```
Comment (ccp92) [inline katas/mumbling/README.md]: Might be worth expanding on the rules of the formatting.
You can deduce it from the example code you provide, but it's better to be explicit.

File: katas/mumbling/README.md (hunk 1 above)
Comment (dushan-madetech) [reply katas/mumbling/README.md]: Amended to match the irb style

File: katas/mumbling/README.md (hunk 2 above)
Comment (dushan-madetech) [reply katas/mumbling/README.md]: have amended it to mumble_letters - I'm not sure exactly what to call it, is this fine?

File: katas/mumbling/README.md (hunk 3 above)
Comment (dushan-madetech) [reply katas/mumbling/README.md]: I've now amended it to provide some clarity on the expected output

**Pull request merged**